build/__variables.sh

```diff
@@ -31,7 +31,9 @@ declare -r BUILD_IMAGES_VSO_DOCKERFILE="$REPO_DIR/images/build/Dockerfiles/vso.D
 declare -r BUILD_IMAGES_BUILDSCRIPTGENERATOR_DOCKERFILE="$REPO_DIR/images/build/Dockerfiles/buildScriptGenerator.Dockerfile"
 declare -r BUILD_IMAGES_GITHUB_RUNNERS_BUILDPACKDEPS_STRETCH_DOCKERFILE="$REPO_DIR/images/build/Dockerfiles/gitHubRunners.BuildPackDepsStretch.Dockerfile"
 declare -r RUNTIME_BASE_IMAGE_DOCKERFILE_PATH="$RUNTIME_IMAGES_SRC_DIR/commonbase/Dockerfile"
+declare -r RUNTIME_BUSTER_BASE_IMAGE_DOCKERFILE_PATH="$RUNTIME_IMAGES_SRC_DIR/commonbase/buster.Dockerfile"
```

Comment:
is this used anywhere?

Reply:
removed

---
In reply to: [430620208](https://github.com/microsoft/Oryx/pull/592#discussion_r430620208) [](ancestors = 430620208)